tests/hardhat/Fork/borrowAndRepayTest.ts

```diff
@@ -96,7 +98,8 @@ if (FORK) {
       await configureTimelock();
       acc1Signer = await initMainnetUser(ACC1, ethers.utils.parseUnits("2"));
       acc2Signer = await initMainnetUser(ACC2, ethers.utils.parseUnits("2"));
-      acc3Signer = await initMainnetUser(ACC3, ethers.utils.parseUnits("2")); // it will be the depositor
+      acc3Signer = await initMainnetUser(ACC3, ethers.utils.parseUnits("2"));
```

Comment:
I think we can remove `acc3Signer` and `ACC3` variables. They don't participate in the tests

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/360/commits/c0e32f76f0e89ca44b46a0eb0a95ca14d27537a4